annif/cli.py

```diff
@@ -123,6 +123,28 @@ def run_show_project(project_id):
     click.echo(template.format('Access:', proj.access.name))
 
 
+@cli.command('init')
+@click.argument('project_id')
+@common_options
+def run_init_project(project_id):
+    """
+    Remove the data files of a project.
+    """
+
+    proj = get_project(project_id)
+    datadir_path = proj.datadir
+    data_files = os.listdir(datadir_path)
+    if data_files:
+        for data_file in data_files:
```

Comment:
Would it be easier and/or cleaner to just remove the entire directory? That would be exactly the same as the original state.

Reply:
True, naturally :) The reason for not-so-straightforward previous approach was that already accessing `proj.datadir` [created the directory](https://github.com/NatLibFi/Annif/blob/master/annif/datadir.py#L14), and raising error message for trying to init an already empty project needed the check of _file_ existence in the directory. 

But now the directory path is got using `proj._datadir_path`, simplifying the necessary logic.

Reply:
Hmm. I don't like accessing a "private" field like this. I'd rather move the init code to the AnnifProject class and just call it from the CLI code.